src/hotspot/cpu/aarch64/aarch64.ad

```diff
@@ -16167,7 +16189,9 @@ instruct CallLeafNoFPDirect(method meth)
 // Also known as an 'interprocedural jump'.
 // Target of jump will eventually return to caller.
 // TailJump below removes the return address.
-instruct TailCalljmpInd(iRegPNoSp jump_target, inline_cache_RegP method_ptr)
+// Don't use rfp for 'jump_target' because a MachEpilogNode has already been
+// emitted just above the TailCall and it will reset rfp to the caller state.
```

Comment:
```suggestion
// emitted just above the TailCall which has reset rfp to the caller state.
```

Reply:
Thanks. I updated the comments accordingly.